tests/test_model_generic.py

```diff
@@ -36,9 +36,8 @@ def test_model_setget_attr():
         a.name = 'redefined-apple'
 
 
-
-def test_model_setattr(pk, kube):
```

Comment:
I should've added the kube fixture everywhere but I still think that we should use it for namespaced models. it adds another dev dependency though...